react/features/filmstrip/components/web/ThumbnailBottomIndicators.js

```diff
@@ -11,6 +11,7 @@ import { LAYOUTS } from '../../../video-layout';
 import StatusIndicators from './StatusIndicators';
 
 declare var interfaceConfig: Object;
+declare var config: Object;
```

Comment:
Don' do this please. Use a selector to get the state from /features/base/config

Reply:
Oh, ok, just followed the interfaceConfig example. Will pass it via props 😃